app/view.js

```diff
@@ -417,7 +438,7 @@ CATS.View = Classify({
                 header +
                 "</div>" +
                 "<div id='catsscore_filters_wrapper'>" +
-                    this.page_settings(this.page_name())(this.get_settings_params(params)) +
+                    this.page_settings(this.page_name())(filters) +
```

Comment:
AFAIR `page_settings` is a model, it should not be polluted with styles and icons